File: tuts/011-getting-started-batch-fargate/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { BatchClient } from "@aws-sdk/client-batch";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new BatchClient({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running batch getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();

File: tuts/011-getting-started-batch-fargate/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_batch "github.com/aws/aws-sdk-go-v2/service/batch"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_batch.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running Batch getting started scenario...")
	// TODO: setup, interact, teardown
}
